sklearn/utils/extmath.py

```diff
@@ -332,6 +333,12 @@ def randomized_svd(
             the value of `random_state` explicitly to suppress the deprecation
             warning.
 
+    lapack_svd_driver : {"gesdd", "gesvd"}, default="gesdd"
+        Whether to use the more efficient divide-and-conquer approach
+        (`"gesdd"`) or more general rectangular approach (`"gesvd"`) to compute
+        the SVD of the matrix B, which is the projection of M into a low
+        dimensional subspace, as described by Halko et. al.
```

Comment:
> as described by Halko et. al.

Can you please add a reference to the mention article?

Reply:
I added a reference to the function's docstring, as it's mentioned multiple times during the parameter documentation.

Reply:
Actually, I just checked and  a reference is already in the function.

Reply:
That's right. In that case, can you adapt the doc-string to add a Sphinx cross-reference similarly to this?

https://github.com/scikit-learn/scikit-learn/blob/16419d1cb75eb4e30e16ba6be8ffe07fd3022af8/sklearn/cluster/_optics.py#L203-L211